imports/client/components/styling/constants.tsx

```diff
@@ -1,9 +1,11 @@
-import { Solvedness } from '../../../lib/solvedness';
+import type { Solvedness } from '../../../lib/solvedness';
```

Comment:
No objection, but I'm mildly curious why this changed. (When Typescript added `import type`, I thought about tracking down all the places we were importing types and making them explicit, but couldn't figure out a good reason to)

Reply:
No particularly reason, just me being slightly sloppy.

I was observing vscode complaining about the following, which is totally not a real problem:

<img width="758" alt="image" src="https://user-images.githubusercontent.com/307325/150654602-53ca76ba-928b-4cca-9007-22034935539a.png">

So I tried switching it to just importing the type instead.  But that didn't appear to have the effect I was hoping for either.  Restarting vscode appears to have cleared up its confusion, so I bet it just missed dispatching the file creation event in some cache somewhere.